lib/internal/test_runner/test.js

```diff
@@ -362,8 +364,37 @@ class Test extends AsyncResource {
   }
 
   matchesTestNamePatterns() {
-    return ArrayPrototypeSome(testNamePatterns, (re) => RegExpPrototypeExec(re, this.name) !== null) ||
-      this.parent?.matchesTestNamePatterns();
+    const matchesByNameOrParent = ArrayPrototypeSome(testNamePatterns, (re) =>
+      RegExpPrototypeExec(re, this.name) !== null ||
+      this.parent?.matchesTestNamePatterns(),
+    );
```

Comment:
```suggestion
    const matchesByNameOrParent = ArrayPrototypeSome(testNamePatterns, (re) =>
      RegExpPrototypeExec(re, this.name) !== null
    )  ||
      this.parent?.matchesTestNamePatterns();
```
just found this issue. can you fix?

Reply:
fixed 👍🏼 

very good catch, such a sneaky mistake, thank you 🙇🏼  or let's blame Copilot 😄